scene/2d/tile_map.cpp

```diff
@@ -1201,9 +1251,21 @@ void TileMap::_bind_methods() {
 	ObjectTypeDB::bind_method(_MD("set_collision_layer", "mask"), &TileMap::set_collision_layer);
 	ObjectTypeDB::bind_method(_MD("get_collision_layer"), &TileMap::get_collision_layer);
 
+	ObjectTypeDB::bind_method(_MD("set_layer_mask", "mask"), &TileMap::set_layer_mask);
+	ObjectTypeDB::bind_method(_MD("get_layer_mask"), &TileMap::get_layer_mask);
```

Comment:
Thank you, I didn't know I could bind two GDscript methods to one C++ method.
I will change it tonight (French time).
Should I make a 2nd commit or use "git rebase-i to mark as a fixup?